eng/Version-Increment.ps1

```diff
@@ -0,0 +1,90 @@
+<#
+.SYNOPSIS
+Bumps up package versions after release
+
+.DESCRIPTION
+This script bumps up package versions following conventions defined at https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#incrementing-after-release-net
+
+.PARAMETER RepoRoot
+The Root of the repo
+
+.PARAMETER ServiceDirectory
+The Name of the Service Directory
+
+.PARAMETER PackageName
+The Name of the Package
+
+.EXAMPLE
+Updating package version for core
+
+Version-Increment.ps1 -PackageCsprojPath "C:\Git\azure-sdk-for-net\sdk\core\Azure.Core\src\Azure.Core.csproj"
+
+#>
+
+[CmdletBinding()]
+Param (
+    [Parameter(Position=0)]
+    [ValidateNotNullOrEmpty()]
+    [string] $RepoRoot,
+    [string] $ServiceDirectory,
```

Comment:
Do we need to always provide this? I'd prefer to only pass in the package name. Keep in mind we want to plug this into https://dev.azure.com/azure-sdk/internal/_git/azure-sdk-build-tools?path=%2Fpipelines%2Fstages%2Farchetype-net-release.yml&version=GBmaster&line=341&lineEnd=342&lineStartColumn=1&lineEndColumn=1&lineStyle=plain and so make sure we have all the pieces we need to pass to this.

Reply:
We have the `ServiceDirectory` passed in along with the other parameters. Except for core which passes '*'. Without the service directory we will have to look through each service folder looking for the package. Or is there another way to go about finding the needed csproj

Reply:
OK as part of enabling this then we probably should fix the "*" service directory problem and have ServiceDirectory always be set to the service directory path and if we need a "*" for some steps we should set that as another property. Take a look at what we did with JS as I think @KarishmaGhiya already did something similar.

Reply:
Will fix that with another PR.